src/components/sections/Testimonials.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./Testimonials.css";
+import Carousel from "react-bootstrap/Carousel";
+import Image from "react-bootstrap/Image";
+
+interface Testimonial {
+  name: string;
+  img: string;
+  quote: string;
+}
+
+const TestimonialCard = ({ name, img, quote }: Testimonial) => {
+  return (
+    <div className="d-flex flex-column align-items-center testimonial-card px-5 pb-3 bg-white mx-auto">
+      <span className="quote-symbol pink-500 font-weight-bold mb-1">“</span>
+      <span className="quote-text mb-3">{quote}</span>
+
+      <div className="d-flex flex-row align-items-center">
+        <Image
+          className="small-img"
+          src={img}
+          alt="Testimonial"
+          roundedCircle
+        />
+        <span className="text-break gray-500 ml-3">{name}</span>
+      </div>
+    </div>
+  );
+};
+
+const Testimonials: React.FC = () => {
+  const testimonials: Testimonial[] = [
+    {
+      name: "Shari Copeland",
+      img: require("../../assets/testimonials/shari.png"),
```

Comment:
is it possible to import this at the top of the file?

Reply:
one